services/sqs.cfc

```diff
@@ -28,19 +28,240 @@ component {
     
     /**
     * Sends a message
-    * @queueName the name of the queue to send to (e.g. "123456789/my-sqs-queue").
-    * @message the message to post, text format.
+    * https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html
+    * @queueName Required string. The name of the queue to send to (e.g. "123456789/my-sqs-queue").
+    * @message Required string. The message to post, text format.
+    * @delaySeconds Optional numeric. The length of time, in seconds, for which to delay a specific message. 
+    * Valid values: 0 to 900. Maximum: 15 minutes. Messages with a positive DelaySeconds value become available for 
+    * processing after the delay period is finished. If you don't specify a value, the default value for the queue applies.
+    * Note: When you set FifoQueue, you can't set DelaySeconds per message. You can set this parameter only on a queue level.
+    * @messageAttributes Optional array. An array of message attributes to be added to the message. Each array item must contain a structure
+    * containing 3 keys: Name, Value, DataType. String and Number are supported DataType values.
+    * Example: [{'Name':'AttName','Value':'AttVal','DataType':'String'},{'Name':'AttName3','Value':34,'DataType':'Number'}]
+    * @messageDeduplicationId Optional string. This parameter applies only to FIFO (first-in-first-out) queues. 
+    * The token used for deduplication of sent messages. 
+    * @messageGroupId Optional string. This parameter applies only to FIFO (first-in-first-out) queues. 
+    * Note: MessageGroupId is required for FIFO queues. You can't use it for Standard queues.
     */
     public any function sendMessage(
+            required string queueName,
+            required string message,
+            numeric delaySeconds,
+            array messageAttriubes = [ ],
```

Comment:
Looks like there is a typo here.

Reply:
Thanks for catching that, I've fixed it.